app/backend/lib/emails/templates/assessmentAssigneeChange.ts

```diff
@@ -78,7 +83,7 @@ const assessmentAssigneeChange: EmailTemplateProvider = async (
           const alerts = (assignments as Array<any>).map((assignment) => {
             return {
               url: `${url}/analyst/application/${assignment.applicationId}/assessments/${assignment.assessmentType}`,
```

Comment:
The url here for financial risk gives a 404, in my code right now I've added the following constant: 
```js
const ASSESSMENT_URL_SLUG = {
  technical: 'technical',
  financialRisk: 'financial-risk',
  projectManagement: 'project-management',
};
```
and updated the url there to use the type as a key

Reply:
Since we are using the same slugs and names in multiple templates, took it out to a constant file.

Reply:
Awesome, also appreciate you adding the assessment type for projectManagement and the rest of them